lively.ide/studio/top-bar.cp.js

```diff
@@ -12,6 +12,14 @@ import { SystemTooltip } from 'lively.morphic/tooltips.cp.js';
 import { RichTextPlugin } from '../text/rich-text-editor-plugin.js';
 import { WorldMiniMap } from '../world-mini-map.cp.js';
 
+function nyi (msg, strict = false) {
+  let error = new Error(`Not yet implemented ${msg}`);
```

Comment:
I think personally I'd prefer switching the message around so that, e.g., in this case it would read: "Component Browser not yet implemented". No hard feeling here though, just a thought.

Reply:
You are very right, it was slightly copy-pasta from elsewhere, I'll fox 🦊